### DISCUSSION TITLE
建议开发者平台的DTK入门教程使用apt源内的dev包

### DISCUSSION THREAD

**Discussion by ZhongYic00** [linuxdeepin/developer-center#3318] [Q&A | 问答板块] [answered]
如题，apt就是为了方便分发的，我觉着比从git仓库手动编译用起来方便。
直接拉取git仓库手动编译容易出现版本过新和稳定版不兼容的情况，最好只在内部使用。

**Comment by BLumia** [comment]
~~有这个计划，会以 `-doc` 包的形式提供 html 和 qch 文档，后者可供在 QtCreator 里直接使用。~~

我好像理解错你的意思了 :joy:

**Comment by BLumia** [comment, marked as answer]
看了下，现有文档目前还是沿用的来自 UOS 的旧文档，文档里甚至也还有关于激活系统之类的描述。相关开发文档后续应该会更新后以新的形式提供。

cc @myml
